tests/backingTypes.spec.ts

```diff
@@ -59,7 +59,10 @@ describe('backingTypes', () => {
             source: path.join(__dirname, '_types.ts'),
           },
         ],
-        contextType: 't.TestContext',
+      },
+      contextType: {
+        path: path.join(__dirname, '_types.ts'),
+        name: 'TestContext',
```

Comment:
Wondering if the type of `contextType` shouldn't be instead:

```ts
{
  module: string // instead of path
  name: string // name could be `export` here
}
```

Having `module` would be more consistent with the next PR which will rename `typegenAutoConfig.sources` `sourceTypes.modules`

Reply:
Agreed, and JS doc will make it clear that it is a path.

I personally find `name` not good either. Would suggest `import` instead.

Verbose would be `modulePath` and `importName`. But I think `module`/`import` are enough.

Reply:
It could be argued that we should use the import syntax as inspiration

```ts
import { foo } from 'bar'
```

```ts
{
  import: 'foo',
  from: 'bar'
}
```

:)

Reply:
Actually I think this might be best:

```ts
{
  module: 'bar'
  export: 'foo',
}
```

Since its specifying how to import, which is different than being an import.

```ts
import { foo } from 'bar'
//                   ^the module to import from
//       ^the export to import
```

:)

Reply:
done 👍